styles.css

```diff
@@ -527,3 +527,102 @@ body {
   bottom: 74.49%;
   opacity: 0.5;
 }
+
+.contact-section {
+  position: absolute;
+  width: 100%;
+  height: 813px;
+  left: 0;
+  top: 4288px;
+  background: #f9f9fa;
+}
+
+.contact-form {
+  padding: 0 20px;
+}
+
+.contact-form h3 {
+  font-family: "Playfair Display", Helvetica, sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 50px;
+  line-height: 100%;
+  color: #1a2236;
+  padding-bottom: 10px;
+}
+
+.contact-form p {
+  font-family: "Poppins", Helvetica, sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 14px;
+  line-height: 21px;
+  letter-spacing: -0.01em;
+  color: #1a2236;
+}
+
+.form-area {
+  text-align: center;
+  margin: 0 auto;
+}
+
```

Comment:
- [ ] Kindly make sure your design looks as close to the Figma design as possible. 🙏 

Figma Design | Your Design
------------- | -------------
![image](https://user-images.githubusercontent.com/74814780/177424962-7da86bd5-f39d-4c82-bfd6-9a0e0b8d90d8.png)  |  ![image](https://user-images.githubusercontent.com/74814780/177424983-caa8f7b8-bc94-41e7-ae0a-1dea75b51182.png)

Reply:
Fixed